Tests/MMEEventTests.m

```diff
@@ -66,7 +66,7 @@ - (void)testNSSecureCodingOfMMEEvent {
      MMEEvent *unarchived = [unarchiver decodeObjectOfClass:MMEEvent.class forKey:NSKeyedArchiveRootObjectKey];
 
      XCTAssertNotNil(unarchived);
-     XCTAssert(unarchived == event);
+     XCTAssertEqualObjects(unarchived, event);
```

Comment:
Test Fix] testNSSecureCodingOfMMEEvent

Problem:
Comparison of objects fails

Why:
original vs unarchived failed b/c we are comparing the address of two objects which will be different. For object comparison, Asserts must use EqualObjects for expected comparison.

Change:
Make comparison with XCTAssertEqualObjects